sycl/test-e2e/NonUniformGroups/chunk.cpp

```diff
@@ -0,0 +1,165 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+//
+// CPU AOT targets host isa, so we compile on the run system instead.
+// RUN: %if any-device-is-cpu && opencl-aot %{ %{run-aux} %clangxx -fsycl -fsycl-targets=spir64_x86_64 -o %t.x86.out %s %}
+// RUN: %if cpu && opencl-aot %{ %{run} %t.x86.out %}
+//
+// REQUIRES: cpu || gpu
```

Comment:
I'm not sure if I completely understand this, but we want to run this test on a CPU with `opencl-aot` LIT feature available, right?
If so, can we do `REQUIRES: cpu && opencl-aot` instead? and remove `%if`?

Reply:
I honestly don't understand the comment there. I think it would be better to just remove the comment.

Previously we had a `REQUIRES: opencl-aot`, but that seemed to be a misunderstanding of how that worked. Now it is just that we test AOT for OCL CPU when possible.